src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -648,6 +649,11 @@ public void postBuildStatus(@NonNull BitbucketBuildStatus status) throws IOExcep
         postRequest(url, JsonParser.toJson(status));
     }
 
+    @Override
+    public BitbucketBuildStatuses getBuildStatus(@NonNull String hash) throws IOException, InterruptedException {
+        return null;
+    }
+
```

Comment:
And here it does return null, causing an exception in the caller. This should preferably be implemented, or at least the caller should check for null.